nvflare/apis/fl_constant.py

```diff
@@ -552,3 +554,27 @@ class RunnerTask:
     INIT = "init"
     TASK_EXEC = "task_exec"
     END_RUN = "end_run"
+
+
+class ConnPropKey:
+
+    PROJECT_NAME = "project_name"
+    SERVER_IDENTITY = "server_identity"
+    IDENTITY = "identity"
+    PARENT = "parent"
+    FQCN = "fqcn"
+    URL = "url"
+    SCHEME = "scheme"
+    ADDRESS = "address"
+    CONNECTION_SECURITY = "connection_security"
+
+    RELAY_CONFIG = "relay_config"
+    CP_CONN_PROPS = "cp_conn_props"
+    RELAY_CONN_PROPS = "relay_conn_props"
+    ROOT_CONN_PROPS = "root_conn_props"
+
+
+class ConnectionSecurity:
```

Comment:
It's complicated to define ConnectionSecurity with relays. We should enforce the same level between all connections. 

mtls gets complicated because we can't reliably pass client cert to the destination with relay in the middle. I suggest we don't support mtls with relays.

Reply:
We need to support connection security for relay too because relays, like the Server, are also access points for clients.